### ISSUE TITLE
Update for Pester v5

### ISSUE THREAD

**Issue by fabiopennisifmg** [jpsider/Invoke-Automation#20]
Hey,

Used your great work to scaffold a new module, thanks!

I have noticed though that it does not seem to work with the latest version of Pester. There's been a few changes in how you call invoke-pester (v5), see here https://pester-docs.netlify.app/docs/quick-start

I will have a crack my end, but don't want to miss anything, so thought I would flag it here.

**Comment by jpsider** [comment]
Yeah, pester 5 has rocked my world! It's a huge change, all the previous modules I've built will not work with v5 yet.

I need to just bite the bullet and learn it, then update all my code. I imagine it's nothing too bad, just going to take some time!

If you need an example of the v4 override, let me know.
